hermes/models.py

```diff
@@ -1,3 +1,16 @@
+from pydoc_data.topics import topics
 from django.db import models
 
-# Create your models here.
+
+class Message(models.Model):
+    topic = models.TextField(blank=True)
+    title = models.TextField(blank=True)
```

Comment:
is there a danger of having these be text fields with no character limit when this becomes publicly accessible from an API?